packages/react/components/skeleton/src/Skeleton.types.ts

```diff
@@ -0,0 +1,11 @@
+import { IContentLoaderProps } from 'react-content-loader';
+import type { CSS } from '@project44-manifest/react-styles';
+
+export type SkeletonElement = 'svg';
```

Comment:
`svg` here is insufficient for Typescript as the `ContentLoader` from `react-content-loader` accepts other props like uniqueKey, speed, etc, but unsure how to type this. 

Tried `typeof ContentLoader` 
<img width="1423" alt="image" src="https://user-images.githubusercontent.com/3459902/222252593-d01e0b6f-7d21-42f2-ae16-58bc70ad1849.png">